boot/seal.go

```diff
@@ -155,6 +155,10 @@ type SealKeyForBootChainsParams struct {
 	RecoveryBootChains []BootChain
 	// RoleToBlName maps bootloader role to the name of its bootloader
 	RoleToBlName map[bootloader.Role]string
+}
+
+type SealKeyForBootChainsParams struct {
+	*BootChains
```

Comment:
why a pointer? could potentially panic? same comment for `ResealKeyForBootChainsParams`.